modules/monitoring/library/Monitoring/Web/Rest/RestRequest.php

```diff
@@ -209,6 +216,7 @@ public function send()
             "Host: {$url['host']}",
             "Content-Type: {$this->contentType}",
             'Accept: application/json',
+            'Connection: keep-alive',
```

Comment:
Imho this shouldn't be hardcoded in this generic class, but left to the caller (e.g. the command transport).

Reply:
@dnsmichi Some other headers *are* already hardcoded – so what.

Reply:
It is a behavioural change. I wouldn't go with that argument of other default headers also being hardcoded but rather make it optional.